### DISCUSSION TITLE
How do I invite users?

### DISCUSSION THREAD

**Discussion by emanaton** [Leantime/leantime#651] [Q&A]
I can see that I can create users, assigning them a name and password, but how do I invite users to login, and to create a username and password when they do so?

**Comment by marcelfolaron** [comment]
When you create a new user Leantime will send an email to the email address
with the password (we are changing this flow to send the user to create a
new password).

Are you asking to just send an invitation without creating the user
upfront? We have been working on a registration page to let users register
them selfs and be assigned to a default project. Would that work?

**Reply by emanaton** [reply]
Thank you for the prompt reply, @marcelfolaron -- and on a weekend, no less! <3

For my purposes, I'm keen to not be responsible for assigning passwords, and/or for users to be able to to use a password I assign more than once. So either of the solutions you've outlined would work for my needs, _or_ if a password could be set to expire after its first use (i.e. so a use could be created and assigned a password like "ChangeThisPasswordRightNowPlease" that would only be good for one login); this might be a rephrasing of what you already outlined though?

Do you know if there's a timeline for when that first solution you mentioned is expected to be available?

Thanks again!

**Comment by marcelfolaron** [comment]
I'm working on it as we speak.
Should be out by Tuesday at the latest.

Best,
Marcel

**Reply by emanaton** [reply]
Wow! Outstanding! Thank you!

**Reply by albjunior** [reply]
It's ready?

What's the link?

**Comment by Anthrh3x** [comment]
Maybe if you can share information about setting up mail system that can deliver mail to user for registration, that would also be helpful here.

**Reply by TimexPeachtree** [reply]
Initially configure the SMTP settings -  

[Leantime Config SMTP](https://github.com/Leantime/docs/blob/master/installation/configuration.md#smtp-configuration)

After creating users they will receive an email from the system.
